boot/bootutil/src/bootutil_public.c

```diff
@@ -51,22 +51,34 @@
 #endif
 
 #include "bootutil/boot_public_hooks.h"
+#include "bootutil_priv.h"
 
 #ifdef CONFIG_MCUBOOT
 BOOT_LOG_MODULE_DECLARE(mcuboot);
 #else
 BOOT_LOG_MODULE_REGISTER(mcuboot_util);
 #endif
 
-const uint32_t boot_img_magic[] = {
-    0xf395c277,
-    0x7fefd260,
-    0x0f505235,
-    0x8079b62c,
+#if BOOT_MAX_ALIGN == 8
```

Comment:
So far MCUboot defined same magic for all allowed BOOT_MAX_ALIGN values.
What is the reason behind changing that?

Reply:
The `BOOT_MAX_ALIGN` was not configurable, so it always assumed maximum write alignment of 8 bytes.
This PR now allows it to be configurable with a range of values.

Selecting any value other than 8 will change the layout of the image trailer, so a new magic value is required in order for the bootloader to be able to identify the new layout.